app/logic/util.js

```diff
@@ -1,5 +1,27 @@
+firebase = require('firebase');
+
 module.exports = {
   /**
+   * Extract the items from the firebase backend using the userId
+   * 
+   * @param {String} userId
+   */
+  getGroceriesForUser: function(userId) {
+      var ref = new Firebase("https://socobo-dev-project.firebaseio.com/grocery-list/" + userId);
```

Comment:
That is correct. It made sense to me to create a separate entry in the database so the access can be more strictly limited.